### PR TITLE
Fix typos and spacing

### PR DESCRIPTION
**Pull request by gazayas**
1. Fixed a few typos in `lib/rbmysql/charset.rb`
2. Fixed the spacing of the last method in `lib/anemone/core.rb`

I know these are super minor things, especially No. 2, but I thought I'd fix them.
Concerning No. 1, I also changed the sentence a little at line No. 220 (It talks about using the charset encoding that is returned when called the method #encoding which is right before it)

### CODE REVIEW

**Comment by bwatters-r7** [comment]
Jenkins test this please

**Comment by gazayas** [comment]
I fixed the test issue 👍

File: lib/anemone/core.rb [hunk 1]
```diff
@@ -298,10 +298,10 @@ def skip_link?(link)
     #
     # Kills all active threads
     #
-  def shutdown
-      @tentacles.each {|t| t.kill rescue nil }
-      @pages = nil
-  end
+    def shutdown
+        @tentacles.each {|t| t.kill rescue nil }
```
Comment (jmartin-r7) [inline lib/anemone/core.rb]: Sorry to nit pick, only lines 301 & 304 should have shifted.

[From CONTRIBUTING.md](https://github.com/rapid7/metasploit-framework/blob/master/CONTRIBUTING.md#code-contributions)  
* Do stick to the [Ruby style guide](https://github.com/rubocop-hq/ruby-style-guide#source-code-layout)

```
  def shutdown
    @tentacles.each {|t| t.kill rescue nil }
    @pages = nil
  end
```

**Comment by wchen-r7** [comment]
This is a good git exercise for Shelby, I'll hold it and we will try to land this tomorrow. Thank you.

File: lib/anemone/core.rb (hunk 1 above)
Comment (gazayas) [reply lib/anemone/core.rb]: Sorry about that!

**Comment by gazayas** [comment]
Shelby?

**Pull request merged**